ui/webui/src/components/AnacondaHeader.jsx

```diff
@@ -66,7 +67,11 @@ export const AnacondaHeader = ({ beta, title, reportLinkURL, isConnected }) => {
     return (
         <PageSection variant={PageSectionVariants.light}>
             <Flex spaceItems={{ default: "spaceItemsSm" }} alignItems={{ default: "alignItemsCenter" }}>
-                <span className="logo" />
+                <img
+                  src="./logo.svg"
+                  className={"logo " + (logoLoaded ? "loaded" : "")}
+                  onLoad={() => setLogoLoaded(true)}
+                />
```

Comment:
This feels rather hacky, an alternative could be to check if the logo has an actual size in our test? As this is only for tests right?

Reply:
I don't know how it's possible to do that with CSS selectors. :(